src/routes/AuthLayoutRouter/AuthLayoutRouter.js

```diff
@@ -81,12 +81,16 @@ export default function AuthLayoutRouter (props) {
     `/(.*)/${POST_DETAIL_MATCH}`,
     `/(.*)/${GROUP_DETAIL_MATCH}`
   ])
+
+  const paramPostId = matchPath(location.pathname, [
```

Comment:
I wonder if in this case since we are using this again we can use @lorenjohnson suggestion of just .*/${POST_DETAIL_MATCH}?

Reply:
I tried several ways of doing this and it would match only on one of the lines, and thus either give me the postId or the groupSlug, and as a result was causing other parts of the component to error